packages/manager/apps/pci-ai-tools/src/configuration/tracking.constants.ts

```diff
@@ -1,2 +1,153 @@
-export const APP_TRACKING_PREFIX = 'PublicCloud::ai::notebooks';
+import ai from '@/types/AI';
+import { PrivacyEnum } from '@/types/orderFunnel';
+
+export const APP_TRACKING_PREFIX = 'PublicCloud::ai_machine_learning';
+export const NOTEBOOK_TRACKING = `${APP_TRACKING_PREFIX}::ai_notebooks`;
+export const DEPLOY_TRACKING = `${APP_TRACKING_PREFIX}::deploy`;
+export const TRAINING_TRACKING = `${APP_TRACKING_PREFIX}::ai_training`;
 export const PCI_LEVEL2 = '86';
+export const DISCOVERY_PLANCODE = 'project.discovery';
+
+export const TRACKING = {
+  notebooks: {
+    listing: {
+      createNotebooksClick: () =>
+        `${NOTEBOOK_TRACKING}::page::button::create_ai_notebooks`,
+      stopNotebooksDataGridClick: () =>
+        `${NOTEBOOK_TRACKING}::datagrid::button::stop_ai_notebooks`,
+      startNotebooksDataGridClick: () =>
+        `${NOTEBOOK_TRACKING}::datagrid::button::start_ai_notebooks`,
+      deleteNotebooksDataGridClick: () =>
+        `${NOTEBOOK_TRACKING}::datagrid::button::delete_ai_notebooks`,
+      manageNotebooksDataGridClick: () =>
+        `${NOTEBOOK_TRACKING}::datagrid::button::manage_ai_notebooks`,
+      DatagridLinkClick: (columnId: string) =>
+        `${NOTEBOOK_TRACKING}::link::details_${columnId}`,
+    },
+    funnel: {
+      createNotebookConfirmClick: (
+        region: string,
+        ressourceType: ai.capabilities.FlavorTypeEnum,
+        framework: string,
+        editor: string,
+        access: PrivacyEnum,
+        resourcesQuantity: number,
+        priceHourly: any,
+      ) =>
+        `${NOTEBOOK_TRACKING}::funnel::create_ai_notebooks_confirm::datacenter_${region}::ressource_type_${ressourceType}::framework_${framework}::editor_${editor}::access_${access}::resourcesQuantity_${resourcesQuantity}::price_hourly_${priceHourly}`,
```

Comment:
Following last mail by tracking should be : 
```suggestion
      createNotebookConfirmClick: (
        region: string,
        ressourceType: ai.capabilities.FlavorTypeEnum,
        framework: string,
      ) =>
        `${NOTEBOOK_TRACKING}::funnel::create_ai_notebooks_confirm::datacenter_${region}::ressource_type_${ressourceType}::framework_${framework}`,
```